diff --git a/package/actions/infiniteScrollAction/index.js b/package/actions/infiniteScrollAction/index.js
index 5a175f44fab9cffb31af5355f2fd4d3200ce2c3b..643b84ff2aeb8dd58158e5b70d90723971f7db5d 100644
--- a/package/actions/infiniteScrollAction/index.js
+++ b/package/actions/infiniteScrollAction/index.js
@@ -57,7 +57,7 @@ function getScrollEventTarget(element) {
         }
         currentNode = currentNode.parentElement;
     }
-    return window;
+    return document.documentElement;
 }
 function getVisibleHeight(element) {
     if (element === window) {